doc-files/release-notes-20.md

```diff
@@ -0,0 +1,135 @@
+[JDK-8295236](https://bugs.openjdk.org/browse/JDK-8295236) | Update JavaDoc in javafx.geometry.Point3D | graphics
+[JDK-8295324](https://bugs.openjdk.org/browse/JDK-8295324) | JavaFX: Blank pages when printing | graphics
+[JDK-8295327](https://bugs.openjdk.org/browse/JDK-8295327) | JavaFX - IllegalArgumentException when printing with margins equal to 0 | graphics
+[JDK-8296854](https://bugs.openjdk.org/browse/JDK-8296854) | NULL check of CTFontCopyAvailableTables return value is required | graphics
+[JDK-8297554](https://bugs.openjdk.org/browse/JDK-8297554) | Remove Scene.KeyHandler | graphics
+[JDK-8297680](https://bugs.openjdk.org/browse/JDK-8297680) | JavaDoc example for PseudoClass has minor typo | graphics
+[JDK-8293971](https://bugs.openjdk.org/browse/JDK-8293971) | Loading new Media from resources can sometimes fail when loading from FXML | media
+[JDK-8294400](https://bugs.openjdk.org/browse/JDK-8294400) | Provide media support for libavcodec version 59 | media
+[JDK-8297362](https://bugs.openjdk.org/browse/JDK-8297362) | EOS might not be delivered by progressbuffer in some cases | media
+[JDK-8293587](https://bugs.openjdk.org/browse/JDK-8293587) | Fix mistakes in FX API docs | other
+[JDK-8295962](https://bugs.openjdk.org/browse/JDK-8295962) | Reference to State in Task.java is ambiguous when building with JDK 19 | other
+[JDK-8303019](https://bugs.openjdk.org/browse/JDK-8303019) | cssref.html incorrect internal link in Path | other
+[JDK-8279214](https://bugs.openjdk.org/browse/JDK-8279214) | Memory leak in Scene after dragging a cell | scenegraph
+[JDK-8297130](https://bugs.openjdk.org/browse/JDK-8297130) | ComboBox popup doesn't close after selecting value that was added with 'runLater' | scenegraph
+[JDK-8300013](https://bugs.openjdk.org/browse/JDK-8300013) | Node.focusWithin doesn't account for nested focused nodes | scenegraph
+[JDK-8222210](https://bugs.openjdk.org/browse/JDK-8222210) | JFXPanel popups open at wrong coordinates when using multiple hidpi monitors | swing
+[JDK-8285881](https://bugs.openjdk.org/browse/JDK-8285881) | Update WebKit to 614.1 | web
+[JDK-8289541](https://bugs.openjdk.org/browse/JDK-8289541) | Update ICU4C to 71.1 | web
+[JDK-8292609](https://bugs.openjdk.org/browse/JDK-8292609) | Cherry-pick WebKit 614.1 stabilization fixes | web
+[JDK-8295755](https://bugs.openjdk.org/browse/JDK-8295755) | Update SQLite to 3.39.4 | web
+[JDK-8298167](https://bugs.openjdk.org/browse/JDK-8298167) | Opacity in WebView not working anymore | web
+[JDK-8292922](https://bugs.openjdk.org/browse/JDK-8292922) | [Linux] No more drag events when new Stage is created in drag handler | window-toolkit
+[JDK-8296621](https://bugs.openjdk.org/browse/JDK-8296621) | Stage steals focus on scene change | window-toolkit
+[JDK-8296654](https://bugs.openjdk.org/browse/JDK-8296654) | [macos] Crash when launching JavaFX app with JDK that targets SDK 13 | window-toolkit
```

Comment:
In the previous release notes, `|` was not wrapped with white-spaces. Do we want to keep it this way?

Reply:
I removed the extra space in all places (even the security bugs section)